ray-operator/config/samples/pytorch-resnet-image-classifier/ray-job.pytorch-image-classifier.yaml

```diff
@@ -88,6 +88,7 @@ spec:
                 image: rayproject/ray-ml:2.9.0-gpu
                 resources:
                   limits:
+                    cpu: "1"
```

Comment:
> Otherwise using this RayJob YAML might result in the error of Pod using more CPU requests than the default limit.

Can you share the exact error for this?

Reply:
I am also interested in this.

Reply:
from our kuberay 1.1.1 logs

`{"level":"info","ts":"2024-06-04T21:55:29.252Z","logger":"controllers.RayCluster","msg":"updateClusterReason","RayCluster":{"name":"pytorch-image-classifier-g9z7q-raycluster-s5dl4","namespace":"hyperkube"},"reconcileID":"a97cf90d-9bee-4cd6-9cdc-89bb04fed89e","Update CR Status.Reason":"Pod \"lassifier-g9z7q-raycluster-s5dl4-worker-gpu-group-hd7ct\" is invalid: spec.containers[0].resources.requests: Invalid value: \"1\": must be less than or equal to cpu limit of 250m"}`

Reply:
Are you using GKE autopilot? It could be autopilot defaulting CPU limits to 250m, although it's a bit surprising that it doesn't default the limit to the request. If you're not using GKE autopilot it could be some other mutating webhook default limits

Reply:
Ah We have a [LimitRange](https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace/#create-a-limitrange-and-a-pod) in that namespace that sets default CPU limit.

Reply:
ah, that would explain it!